src/src_V2/modules/manager/financial/payables/edit-drawer/payment-history/PaymentHistory.jsx

```diff
@@ -199,10 +244,10 @@ export default function PaymentHistory({
           <Table className={styles.historyTable}>
             <TableHead>
               <TableRow>
-                {columnKeys.map((c) => {
+                {columnKeys.map((c, index) => {
```

Comment:
it's minor thing but i think we can change `c` argument here, at lest column or item. Hm?